### DISCUSSION TITLE
Custom code to mute chrome while leaving system sound as is

### DISCUSSION THREAD

**Discussion by CSOCSO-FL** [irzyxa/Volume2#331] [Q&A] [answered]
I had a security breach, and one of the countermeasures I came up with is to have windows play a chime when someone logs into windows. I asked chatgpt to make me such code and set it up through task scheduler. I have a wav file that plays every time i log into the windows, but if I listen to music via chrome or VLC I might not hear the chime. I don't want the chime to be crazy loud or alarming that drives me nuts every time I log in, so I want the script to mute the app ( chrome.exe and vlc.exe) while leaving system sound as is, play the chime and put the volume back up after. 
I downloaded volume2 and selected Audio device as system default ( sometimes i listen to music on the normal speakers, sometimes via bt speakers next door)
And set change volume for specific app ( chrome.exe)
Alternative audio is set to No. For some reason while this worked to control the volume of chrome a couple days ago, right now it does not work. Not sure why.  Btw my main issue was that even with this script all the volume was muted not just what the volume2 was controlling.

here is the powershell script if anyone wants to see it or if it helps: ( i took out the hashtags from in front of the sentences)

Define the path to Volume2
$volume2Path = "C:\Program Files (x86)\Volume2\Volume2.exe"

 Mute Chrome by setting its volume to 0
Start-Process -FilePath $volume2Path -ArgumentList "/Mute /AppName ""chrome.exe""" -NoNewWindow -Wait

 Wait for 3 seconds
Start-Sleep -Seconds 3

Unmute Chrome by setting its volume back to a normal level (assuming 100%)
Start-Process -FilePath $volume2Path -ArgumentList "/Unmute /AppName ""chrome.exe""" -NoNewWindow -Wait

**Here is another one chatgpt gave me before:**

 **_Define the path to Volume2
$volume2Path = "C:\Program Files (x86)\Volume2\Volume2.exe"

 Set the system volume to 0 (or any other volume level you prefer)
Start-Process -FilePath $volume2Path -ArgumentList "/SetVolume 0" -NoNewWindow -Wait

 Wait for 3 seconds
Start-Sleep -Seconds 3

 Restore the system volume to the previous level (assuming it was 100%)
Start-Process -FilePath $volume2Path -ArgumentList "/SetVolume 100" -NoNewWindow -Wait

Yes Volume2's path is correct.

**Comment by irzyxa** [comment, marked as answer]
Okay, Volume2 does not support /APP chrome.exe. But you can use alternative device, set control specific application chrome.exe and then use this parameter: /MUTE ALTDEVICE
